app/containers/CustomFields/index.tsx

```diff
@@ -12,10 +12,16 @@ interface ICustomFields {
 	customFieldsRef: React.MutableRefObject<{
 		[key: string]: TextInput | undefined;
 	}>;
-	onSubmit: () => void;
+	onSubmit?: () => void;
```

Comment:
`onSubmit` should be required, so we're forced to keep navigation through form items working